### DISCUSSION TITLE
2.2.0 commit from Grisha Naychenko supply chain hack or legit?

### DISCUSSION THREAD

**Discussion by kenbailey** [ElmahCore/ElmahCore#187] [General]
We are interested in the filtering options added in the 2.2.0 commits to this project, but they appear suspicious for a number of reasons:

* Grisha Naychenko account no longer exists on GitHub and I don't see any other commits by the individual
* The change includes a change relaxing the CORS settings in ElmahCore\ElmahCore.DemoCore6\Program.cs
* Running the DemoCore6 application on 2.2.0 has excessive CPU usage that goes away as soon as you go back to previous commits
* 2.2.0 has not been build for nuget.org

Can any of the maintainers comment or clear out this commit if it's bogus? The project seemed solid but has went silent.

**Comment by afastrunner** [comment]
I don't know the details but I believe this person is building upon this code and may be the place to look for ongoing development. https://github.com/jrsearles/Elmah.AspNetCore

**Reply by kenbailey** [reply]
Thanks, @afastrunner. Will look into the fork further.
